client/components/back-button/style.scss

```diff
@@ -0,0 +1,41 @@
+.back-button {
+	margin: 0;
+	position: fixed;
+		top: 47px;
+		left: 0;
+	z-index: z-index( '.masterbar', '.reader-back'	);
+
+	.button.is-compact.is-borderless {
+		padding: 18px 40px 18px 20px;
+
+		@include breakpoint( "<660px" ) {
```

Comment:
Small nitpick: I think this should be >660px and have the paddings switched up?

Reply:
Good call. Mobile first 📱